Resources/doc/index.rst

```diff
@@ -58,20 +58,20 @@ Step 2: Enable the bundle
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Enable the bundle in the kernel:
+ 
+.. code-block:: php
```

Comment:
Correct me if I am wrong, I think what you are saying is without syntax highlighting.
What I used is correct syntax for a code block: http://sphinx-doc.org/markup/code.html